content/300-accelerate/900-compare.mdx

```diff
@@ -0,0 +1,106 @@
+---
+
+<TopBlock>
+
+Prisma Accelerate supports products that serve a global audience, with a global caching system and connection pool that spans multiple regions, providing consistent access to data with low latency no matter where your user (or your database) is located in the world.
+
+The managed connection pool is designed to support serverless infrastructure, capable of handling high volumes of connections and adapting to traffic spikes with ease.
+
+Explore how Prisma Accelerate compares to other global cache and connection pool solutions on the market, and discover what sets it apart.
+
+</TopBlock>
+
+## What makes Accelerate unique?
+
+Prisma Accelerate is chosen and loved by many for a number of key reasons which make Accelerate unique:
+
+- [**Query-Level Policies**](/accelerate/compare#accelerate-global-cache): Accelerate is the only solution that offers query-level cache policies, allowing you to control the cache strategy for each query specifically. It is common to have some values that need to be cached for a long time, others that need caching for a short time, and some that should not be cached at all. With Accelerate you can do this, and even set different cache strategies per query.
+- [**Global by default**](/accelerate/compare#accelerate-global-cache): Accelerate is globally distributed by default. You never need to worry about where a user is located with respect to your database location.
+- [**Fully managed**](/accelerate/compare#management): You don't need to manage a server or worry about uptime. Accelerate is fully managed for you.
+- [**Auto-scaling**](/accelerate/compare#performance): Accelerate automatically adjusts resources to match workload demands, providing fast and consistent performance during traffic spikes.
+
+## Accelerate global cache
+
+Prisma Accelerate offers a powerful global cache, so you can serve data to your users at the edge — the closest point to where the users are located — no matter where your database is hosted. This not only speeds up the experience for users, but also reduces read load on your database as well by avoiding roundtrips.
+
+|  | Accelerate | Hyperdrive | Planetscale Boost |
+| --- | --- | --- | --- |
+| **Fully Managed** | ✅ | ✅ | ✅ |
+| **Globally distributed edge infra** | ✅ | ✅ | ✅ |
+| **Control cache policy from code** | ✅ | ❌ | ❌ |
+| **Query-level cache policies** | ✅ | ❌ | ❌ |
+| **Authenticate with API key** | ✅ | ❌ | ❌ |
+| **Postgres compatible** | ✅ | ✅ | ❌ |
+| **MySQL compatible** | ✅ | ❌ | ✅ |
+| **MongoDB compatible** | ✅ | ❌ | ❌ |
+| **Automatic cache updates** | ❌ | ❌ | ✅ |
+
+**Why are these important?**
+
+- Integration with the Prisma client is important for existing Prisma ORM users — it means you can control caching policies directly from your existing codebase with just an extra line of code. Integration is seamless. Here is an example using [the stale-while-revalidating caching strategy](/accelerate/caching#stale-while-revalidate-swr):
```

Comment:
```suggestion
- Since Accelerate extends the Prisma client, you can control caching policies directly from your codebase with just an extra line of code. Integration is seamless. Here is an example using [the stale-while-revalidating caching strategy](/accelerate/caching#stale-while-revalidate-swr):
```